core/src/main/java/org/fao/geonet/kernel/metadata/DefaultStatusActions.java

```diff
@@ -264,6 +264,11 @@ protected void notify(List<User> userToNotify, MetadataStatus status) throws Exc
             textTemplate = messages.getString("status_change_default_email_text");
         }
 
+        // Replace link in message
+        ApplicationContext applicationContext = ApplicationContextHolder.get();
+        SettingManager sm = applicationContext.getBean(SettingManager.class);
+        textTemplate = textTemplate.replace("{{link}}", sm.getNodeURL()+ "api/records/{{index:uuid}}");
+
```

Comment:
@ianwallen with this change, should be updated also this code, it seems no longer required the `metadataUrl`, no?

https://github.com/geonetwork/core-geonetwork/pull/7130/files#diff-e357e9d4b0a5a909d57c7631974e007dd33391912ca34fe1d6d415114bb001a9L273-L279

Reply:
This was done for backward compatibility

If someone is using {7} in the message then it will generate the link using metadataUtils.getDefaultUrl which has the anchor.
If someone uses {link} then it will use non - anchor link. The {link} was used in other emails so I was making it consistent.

If we don't care about backward compatibility then I can remove the usage of {link}, then set 
`String metadataUrl = sm.getNodeURL()+ "api/records/{{index:uuid}}")`

Are you suggesting that we keep using {7} with the non anchor url?

Reply:
Thanks for the clarification, it sounds ok as it is.